task11/index.js

```diff
@@ -0,0 +1,54 @@
+const div = document.getElementById("js-parent");
+
+const loading = () => {
+    const loadingPlace = document.createElement("div");
+    const gif = document.createElement('img');
+    loadingPlace.id = "loadingPlace";
+    gif.src = "loading-circle.gif";
+    div.appendChild(loadingPlace).appendChild(gif);
+};
+
+const removeLoading = () => {
+    document.getElementById("loadingPlace").remove();
+};
+
+const getData = async () => {
+    loading();
+    try {
+        const request = await fetch("https://jsondata.okiba.me/v1/json/3jhPW210812052923");
+        const data = request.json();
+        return data;
+        
+    } catch (error) {
+        div.textContent = "データを読み込めませんでした";
+        console.error(error);
+    }
+    finally{
+        removeLoading();
+    };
+};
+
+const objShow = (value) => {
```

Comment:
objShow ???
Let's name it properly
What is obj??

Reply:
I think so too.This is strange.
I'll be careful.